woocommerce-stats-by-currency.php

```diff
@@ -124,6 +124,26 @@ function wsbc_add_stats() {
 			<?php
 		}
 	}
-	
+
 }
 add_action( 'woocommerce_after_dashboard_status_widget', 'wsbc_add_stats' );
+
+
+// Admin Notice
+add_action( 'admin_init', 'wsbc_required_plugins' );
+
+function wsbc_required_plugins() {
+    if ( is_admin() && current_user_can( 'activate_plugins' ) &&  !is_plugin_active( 'woocommerce/woocommerce.php' )  ) {
```

Comment:
- `admin_init` hooks will only run on admin so `is_admin()` check not required
- only users with `activate_plugins` can activate plugins `current_user_can( 'activate_plugins' )` check here redundant
- We are only checking for `WooCommerce` other dependent plugin `WooCommerce Price By Country` is missing